api_tests/wb/views/test_wb_hooks.py

```diff
@@ -770,7 +771,10 @@ def folder(self, root_node, user):
 
     @pytest.fixture()
     def copy_url(self, preprint):
-        return f'/_/wb/hooks/{preprint._id}/copy/'
+        preprint_id = preprint._id
+        if VersionedGuidMixin.GUID_VERSION_DELIMITER in preprint._id:
+            preprint_id = preprint._id.split(VersionedGuidMixin.GUID_VERSION_DELIMITER)[0]
+        return f'/_/wb/hooks/{preprint_id}/copy/'
```

Comment:
Instead of modifying the test to use base guid, we should update the api code to support versioned guid.